src/main/java/com/google/netpcapanalysis/models/FileAttribute.java

```diff
@@ -0,0 +1,30 @@
+package com.google.netpcapanalysis.models;
+
+import java.util.Date;
+
+public final class FileAttribute {
+
+  public String pcapEntity;
+  public String fileName;
+  public String myIP;
+  public Date uploadDate;
+
+  public FileAttribute() {
+    this(null, null, null, null);
+  }
+
+  public FileAttribute(String pcapEntity, String fileName,String myIP) {
+    this.pcapEntity = pcapEntity;
+    this.fileName = fileName;
+    this.myIP = myIP;
+    this.uploadDate = new Date();
```

Comment:
In the readme you mention this is long, are you converting this to a Epoch Time>?

Reply:
It's stored as Date in datastore, I copied/passed the table and forgot editing that value